.github/workflows/e2e.yml

```diff
@@ -6,19 +6,13 @@ on:
       - master
 
 jobs:
-  tests:
-    if: contains(github.event.comment.html_url, '/pull/') && startsWith(github.event.comment.body, '/run')
-    runs-on: ubuntu-20.04
+  # Job for finding last commit sha of pull-request
+  find-latest-commit-sha:
+    if: contains(github.event.comment.html_url, '/pull/') && startsWith(github.event.comment.body, '/run-e2e')
+    runs-on: ubuntu-latest
+    outputs:
+      commit-sha: ${{ steps.getcommit.outputs.sha }}
```

Comment:
I think this should be enough to checkout on latest commit

```bash
      - uses: actions/checkout@v2
        with:
          ref: ${{ github.event.pull_request.head.sha }}    
```

Reply:
We are using the issue_comment event here, so can't access pull_request.head.sha.